src/mapblock.h

```diff
@@ -601,7 +601,7 @@ typedef std::vector<MapBlock*> MapBlockVect;
 
 inline bool objectpos_over_limit(v3f p)
 {
-	const float max_limit_bs = MAX_MAP_GENERATION_LIMIT * BS;
+	const float max_limit_bs = (MAX_MAP_GENERATION_LIMIT + 0.5f) * BS;
```

Comment:
What is the purpose of this change? Is this to align legal entity and player positions with the edge of the outermost mapblock? If so, why is it part of this PR? It seems like a separate issue to me.

Reply:
> Is this to align legal entity and player positions with the edge of the outermost mapblock?

Yes. Creating a separate PR for this is unnecessary IMO.